### PR TITLE
Add notes on Pluto

### PR DESCRIPTION
**Pull request by taupalosaurus**
I created a pluto.txt file, and put some information about Pluto in it.

### CODE REVIEW

**Comment by mlange05** [comment]
Are you sure it's the biggest? Maybe you want to be a bit clearer.

File: pluto.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Pluto is the biggest planet.
```
Comment (mlange05) [inline pluto.txt]: Really?

**Comment by taupalosaurus** [comment]
Oops my bad. Let me change it.

**Comment by mlange05** [comment]
That's great. Thanks, I'll merge.

**Pull request merged**